.docs/user-guide/troubleshooting.md

```diff
@@ -0,0 +1,241 @@
+```
+Failed to start rexray.service: Interactive authentication required.
+```
+
+### Issue: Leave out service specification on command line, when multiple services are configured:
+`FATA[0000] http error                                    status=404`
+
+### Issue: Place REX-Ray configuration file in wrong place, or fail to create one at all:
+`rexray volume ls` returns:
+`ERRO[0000] error starting libStorage server              error.configKey=libstorage.server.services error.obj=<nil> time=1486771348853`
+
+### Issue: Define bad credentials for the storage provider API
+
+In this case (AWS), results in a log entry like this:
+
+```
+time="2017-02-10T22:34:44Z" level=error msg="error getting volume" host="tcp://127.0.0.1:7979" inner="AuthFailure: AWS was not able to validate the provided access credentials\n\tstatus code: 401, request id: ea6587a90-f29d-4f14-99da-6a7ec7cb05c1" instanceID="ebs=i-0213cc11c4ade43fb,availabilityZone=us-west-2a&region=us-west-2" route=volumesForService server=dew-lady-tw service=ebs storageDriver=ebs task=0 time=1486766084055 tls=false txCR=1486726083 txID=05a5e1fd-094f-40ec-63f6-448d26ddde4f
+```
+
+### Issue: Leave out a service definition, or erroneously name it, in the REX-Ray configuration:
+
+```
+On start ERRO[0000] error starting libStorage server              error.obj=<nil> error.configKey=libstorage.server.services time=1486772035132
+
+In log:
+time="2017-02-11T00:13:25Z" level=error msg="error starting libStorage server" error.configKey=libstorage.server.services error.obj=<nil> time=1486772005732 
+time="2017-02-11T00:13:25Z" level=error msg="default module(s) failed to initialize" error.obj=<nil> error.configKey=libstorage.server.services time=1486772005732 
+time="2017-02-11T00:13:25Z" level=error msg="daemon failed to initialize" error.configKey=libstorage.server.services error.obj=<nil> time=1486772005732 
+time="2017-02-11T00:13:25Z" level=error msg="error starting rex-ray" error.obj=<nil> error.configKey=libstorage.server.services time=1486772005732 
+```
+
+## Logging
+
+Enable detailed logging through the REX-Ray configuration file. Available levels from least detailed, to most are: 
+
+1. panic
+2. fatal
+2. error
+3. warn
+3. info
+4. debug
+
+There is also a setting to enable logging of transactions over the internal REST API.
+
+Also note that for any command line invocation, the `-l debug` flag can be added to get verbose output.
+
+Because REX-Ray is based on a second libStorage component, REX-Ray has two independent log settings in it’s configuration, rexray, and libstorage. For troubleshooting, set the log level for both REX-Ray and libStorage to `debug` and enable HTTP request and response tracing for libStorage. Note that additional lines may be present in your configuration - this example has been simplified to highlight the relevant settings.
+
+REX-Ray config: (`/etc/rexray/config.yml`)
+
+```
```

Comment:
Hi @cantbewong,

Please add `yaml` after the triple-tick to cause the code block to be formatted.